js/main.js

```diff
@@ -97,6 +98,28 @@ function renderEntry(entry) {
 
   return $list;
 }
+
+// edits the entry form when the pencil icon is clicked and opens to Edit Entries form and updates form from data.editing inputs
+function editPencil(event) {
+  for (let i = 0; i < data.entries.length; i++) {
+    if (data.entries[i].entryId === Number(event.target.dataset.entryId)) {
+      data.editing = data.entries[i];
+      const $editTitle = document.querySelector('#title');
+      $editTitle.value = data.editing.title;
+      const $editImg = document.querySelector('#photo-url');
+      $editImg.value = data.editing.imgUrl;
+      const $editImgSrc = document.querySelector('img');
+      $editImgSrc.src = data.editing.imgUrl;
+      const $editNotes = document.querySelector('#notes');
+      $editNotes.value = data.editing.notes;
+      $pageName.textContent = 'Edit Entries';
+      $delButton.setAttribute('class', 'display');
+      data.view = 'entry-form';
```

Comment:
You shouldn't be setting `data.view` here. This should be happening inside of `viewSwap`